src/server/service/kv.rs

```diff
@@ -1186,12 +1186,13 @@ fn future_get<E: Engine, L: LockManager>(
     storage: &Storage<E, L>,
     mut req: GetRequest,
 ) -> impl Future<Output = ServerResult<GetResponse>> {
+    let start = Instant::now();
     let v = storage.get(
         req.take_context(),
         Key::from_raw(req.get_key()),
         req.get_version().into(),
     );
-
+    let used_time = duration_to_ms(start.elapsed());
```

Comment:
The time measurement is not accurate, since variable `v` is an async function call?

Reply:
@crazycs520 you are right. End time should be measured after await.